src/data-access/repositories/mongo/mongo-action-repository.ts

```diff
@@ -27,6 +27,18 @@ export class MongoActionRepository extends BaseMongoRepository implements Action
     return action
   }
 
+  public async getActions(): Promise<Action[]> {
+    this.assertDatabaseConnection(this.getActions.name)
+    return await this.getCollection().find<Action>({ rundownId: { $exists: false } }).toArray()
```

Comment:
Since there is no error handling in this method, the `await` is redundant.

Reply:
Updated